python/pyhail/dataset.py

```diff
@@ -709,6 +986,71 @@ def export_samples(self, output, condition, types=None):
     def export_variants(self, output, condition, types=None):
         """Export variant information to delimited text file.
 
+        **Examples**
+
+        Export a four column TSV with ``v``, ``va.pass``, ``va.filters``, and
+        one computed field: ``1 - va.qc.callRate``.
+
+        >>> vds.export_variants('data/file.tsv',
+        >>>   'VARIANT = v, PASS = va.pass, FILTERS = va.filters, MISSINGNESS = 1 - va.qc.callRate')
+
+        It is also possible to export without identifiers, which will result in
+        a file with no header. In this case, the expressions should look like
+        the examples below:
+
+        >>> vds.export_variants('data/file.tsv', 'v, va.pass, va.qc.AF')
+
+        .. note::
+        Either all fields must be named, or no field must be named.
+
+        In the common case that a group of annotations needs to be exported (for
+        example, the annotations produced by ``variantqc``), one can use the
+        ``struct.*`` syntax.  This syntax produces one column per field in the
+        struct, and names them according to the struct field name.
+
+        For example, the following invocation (assuming ``va.qc`` was generated
+        by :py:meth:`.variant_qc`):
+
+        >>> vds.export_variants('data/file.tsv', 'variant = v, va.qc.*')
+
+        will produce the following set of columns::
+
+            variant  callRate  AC  AF  nCalled  ...
+
+        Note that using the ``.*`` syntax always results in named arguments, so it
+        is not possible to export header-less files in this manner.  However,
+        naming the "splatted" struct will apply the name in front of each column
+        like so:
+
+        >>> vds.export_variants('data/file.tsv', 'variant = v, QC = va.qc.*')
+
+        which produces these columns::
+
+            variant  QC.callRate  QC.AC  QC.AF  QC.nCalled  ...
+
+
+        **Notes**
+
+        This module takes a comma-delimited list of fields or expressions to
+        print. These fields will be printed in the order they appear in the
+        expression in the header and on each line.
+
+        One line per variant in the VDS will be printed.  The accessible namespace includes:
+
+        - ``v`` (variant)
```

Comment:
See annotate_samples_expr for how to format this.

Can link to variant like this:
:ref:`variant`